src/components/book.js

```diff
@@ -1,23 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const Book = ({ books }) => {
+  const dispatch = useDispatch();
+  const deleteBook = (e) => {
+    const btnId = e.target.id;
+    axios
+      .delete(
+        `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/KP5PI7hEkMIt6Rn76UzC/books/${btnId}`,
+      )
```

Comment:
[OPTIONAL]

- You can put the API call in a separate module and call it wherever you need by exporting them. It is very useful when you have to do many API calls in different files.